skimage/registration/_masked_phase_cross_correlation.py

```diff
@@ -12,7 +12,12 @@
 import numpy as np
 from functools import partial
 
-from .._shared.fft import fftmodule, next_fast_len
+from .._shared.fft import next_fast_len
+try:
+    from scipy.fft import fftn, ifftn
+except ImportError:
+    # scipy < 1.4 does not have an fft module
+    from scipy.fftpack import fftn, ifftn

```

Comment:
Can't this take place in `skimage._shared.fft`?

Reply:
There is already a fallback there when `scipy.fft` doesn't exist is `numpy.fft` rather than `scipy.fftpack`. I think that was done there because the behavior of real-to-complex and complex-to-real transforms in `scipy.fftpack` is inconsistent with `scipy.fft` and `numpy.fft`. In this file only complex-to-complex transforms are needed, though, so we can fall back to `scipy.fftpack`.

I can add a TODO to clean this up once we bump the minimum SciPy to 1.4

Reply:
It looks like this fallback doesn't help in any case due to a difference in the `s` argument (CI still fails on the minimum requirements). I may go back to just using the _shared.fft modules and manually fixing the dtype